website/source/docs/providers/aws/d/db_snapshot.html.markdown

```diff
@@ -0,0 +1,65 @@
+---
+layout: "aws"
+page_title: "AWS: aws_db_snapshot"
+sidebar_current: "docs-aws-datasource-db-snapshot"
+description: |-
+  Get information on a DB Snapshot.
+---
+
+# aws\_db\_snapshot
+
+Use this data source to get information about a DB Snapshot for use when provisioning DB instances
+
+## Example Usage
+
+```
+data "aws_db_snapshot" "ebs_volume" {
+    most_recent = true
+    owners = ["self"]
+    db_instance_identifier = "mytestdbinstance"
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `most_recent` - (Optional) If more than one result is returned, use the most
+recent Snapshot.
+
+* `db_instance_identifier` - (Optional) Returns the list of snapshots created by the specific db_instance
+
+* `db_snapshot_identifier` - (Optional) Returns information on a specific snapshot_id.
+
+* `snapshot_type` - (Optional) The type of snapshots to be returned. If you don't specify a SnapshotType 
```

Comment:
`SnapshotType` -> `snapshot_type`

Should also list the allowed values; I assume `manual` and `automated`? Or is it `automatic`?